source/application/PGM/PGMAppController.cpp

```diff
@@ -84,12 +88,27 @@ void PGMAppController::registerExporterOptions()
 
 void PGMAppController::setupScanConfigurations()
 {
-
+	xasScanConfiguration_ = new PGMXASScanConfiguration;
+	xasScanConfiguration_->setControl(0, AMBeamline::bl()->exposedControlByName("Energy")->toInfo());
+//	xasScanConfiguration_->addDetector(AMBeamline::bl()->exposedDetectorByName("teyBladeCurrentDetector")->toInfo());
```

Comment:
I assume these are being left in as we're not yet certain which detector's Lucia is going to require for the XAS scans?